syslogish/server.go

```diff
@@ -16,7 +16,7 @@ const (
 	bindHost          = "0.0.0.0"
 	bindPort          = 1514
 	queueSize         = 500
-	controllerPattern = `^(INFO|WARN|DEBUG|ERROR)\s+(\[(\S+)\])+:(.*)`
+	controllerPattern = `(INFO|WARN|DEBUG|ERROR)\s+(\[(\S+)\])+:(.*)`
```

Comment:
Oh nice, so easy to test and tweak in debuggex. Wow, I really don't know what the deal is.